acceptance/lib/helper.rb

```diff
@@ -28,10 +28,16 @@ def self.initialize_config(options)
     # puppet-agent version corresponds to packaged development version located at:
     # http://builds.delivery.puppetlabs.net/puppet-agent/
     puppet_build_version = get_option_value(options[:puppet_build_version],
-                         nil, "Puppet Development Build Version",
+                         nil, "Puppet Agent Development Build Version",
                          "PUPPET_BUILD_VERSION",
                          "2b6b9db87bb5e6329a5cc6520b6c6db569d4113f", :string)
 
+    # puppetdb version corresponds to packaged development version located at:
+    # http://builds.delivery.puppetlabs.net/puppetdb/
+    puppetdb_build_version =
+      get_option_value(options[:puppetdb_build_version], nil,
+                       "PuppetDB Version", "PUPPETDB_BUILD_VERSION", "3.2.1", :string)
```

Comment:
I pinned to the latest release, but I tested with SNAPSHOT builds too (like today's 4.0.0.SNAPSHOT.2015.11.24T1346) and all is well.